ee/vellum_ee/workflows/display/nodes/vellum/inline_prompt_node.py

```diff
@@ -253,3 +253,33 @@ def _serialize_attributes(self, display_context: "WorkflowDisplayContext"):
                 raise ValueError(f"Failed to serialize attribute '{attribute.name}': {e}")
 
         return attributes
+
+    def _serialize_parameters(self, parameters, display_context: "WorkflowDisplayContext") -> JsonObject:
+        """Serialize parameters, returning empty object when nested descriptors are detected."""
+        from vellum.workflows.descriptors.base import BaseDescriptor
+        from vellum.workflows.references.lazy import LazyReference
+        from vellum.workflows.references.output import OutputReference
+        from vellum.workflows.references.state_value import StateValueReference
+        from vellum.workflows.references.workflow_input import WorkflowInputReference
+
+        params = raise_if_descriptor(parameters)
+        if not params:
+            return {}
+
+        def contains_descriptors(obj):
```

Comment:
Checkout main and rebase this branch. There's a helper that's similar to this defined outside of the serialize method that I want to reuse here.